tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
+    "extends": "@box/frontend/ts/tsconfig",
     "compilerOptions": {
-        "outDir": "./es"
+        "baseUrl": ".",
+        "outDir": "es",
+        "rootDir": "src"
     },
-    "extends": "@box/frontend/ts/tsconfig",
```

Comment:
If paths came directly from the extended config, that will be relative to that config and not this project. tsconfig is missing tokens like <rootDir> that can be used in the path to avoid ambiguity. See https://github.com/microsoft/TypeScript/issues/25430